@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -708,19 +715,15 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
         assert(projection.type !== ProjectionType.Spherical);
         const viewRanges = { ...this.minimumViewRange };
 
-        // Generally near/far planes are set to keep top/bottom planes intersection distance.
-        // Then elevations margins are applied. Here margins (min/max elevations) are meant to
-        // be defined as distance along the ground normal vector thus during camera
-        // tilt they may affect near/far planes positions differently.
+        // Find intersections of top/bottom frustum side's medians with the ground plane, taking
+        // into account min/max elevations.Top side intersection distance determines the far
```

Comment:
While I agree this is a good comment, I don't think the user really cares that much about the specific details

Something simple like "Compute the top / bottom intersection distances for planar" and the rest can go in the comment on the method.

Reply:
done